src/pages/sites/metrics.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: metrics
+path: /sites/metrics
+date: 2020-07-16
+---
+
+import Layout from '../../components/Layout';
+export default ({ children }) => (
+  <Layout title="metrics" description="metrics microsite">
+    <main style={{ maxWidth: `940px`, margin: `0 auto` }}>{children}</main>
+  </Layout>
+);
+
+# nodejs.org Download Metrics
```

Comment:
A margin-top would be nice here as the navbar's height changed in mobile a few days ago

Reply:
Yes, will probably do this in a separate PR.